### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by erikjung**
This is the initial version to be tried on a real project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,23 @@
...
+0. `npm start`
+
+## Overview
+
+Drizzle is a project template and build tool for creating pattern libraries. It outputs static HTML that can live anywhere, using [Gulp], [PostCSS], [Babel], [Webpack] and [Handlebars] under the hood.
+
+## Inspiration
+
+- https://fbrctr.github.io
+- http://solid.buzzfeed.com
```
Comment (tylersticka) [inline README.md:17]: I might also add [Pattern Lab](http://patternlab.io/), which influenced a lot of the changes we requested or made ourselves to Fabricator.

Comment (erikjung) [inline README.md:17]: Will do.

File: src/assets/drizzle/scripts/drizzle.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import 'prismjs';
+
+const dom = {};
+
+dom.nav = document.getElementById('nav');
+dom.navMenu = document.getElementById('nav-menu');
+dom.navToggle = dom.nav.querySelector('a[href="#nav"]');
+dom.navLinks = dom.navMenu.querySelectorAll('a');
+
+function setActiveNavItem (pathname) {
```
Comment (tylersticka) [inline src/assets/drizzle/scripts/drizzle.js:12]: Should we make an issue to create a helper or something to allow us to bake this into the nav, making it more about the current _page_ than the current hash? Just a thought!

Comment (erikjung) [inline src/assets/drizzle/scripts/drizzle.js:12]: I don't quite understand...

I like the idea of using a Handlebars helper (feel free to make an issue for that). Regarding the page vs. hash, I think it already does acknowledge only the page and not the hash (though I definitely haven't tested it thoroughly).

Comment (tylersticka) [inline src/assets/drizzle/scripts/drizzle.js:12]: I'll make an issue and more clearly describe what I mean. This wasn't the relevant place to initiate this discussion. My bad!

**Pull request merged**